_sass/_objects--buttons.scss

```diff
@@ -0,0 +1,17 @@
+$size--horizontal-padding:                $spacer*2.5;
+
+.btn {
+  font-family: $primary-font;
+  cursor: pointer;
+}
+
+.btn-primary {
+  font-size: 22px;
+  font-weight: 600;
+  padding-left: $size--horizontal-padding;
+  padding-right: $size--horizontal-padding;
+
+  &:hover {
+    background-color: $jade;
```

Comment:
Bootstrap variables for hover color, besides, check bootstrap variables for font-size and font-weight (event padding horizontal has its variable like padding-x or something like that) on top of that use font-size as `rem` and using `font-size-base` from bootstrap or a variable that match your need